pkg/flow/reconciler/common/semantic/semantic_test.go

```diff
@@ -173,47 +174,49 @@ func TestServiceEqual(t *testing.T) {
 	}
 
 	for name, tc := range testCases {
-		//nolint:scopelint
+		// //nolint:scopelint
```

Comment:
Don't think we need the second `//` chunk

Reply:
Nice catch!
Done in e1dea361